firebase_admin/project_management.py

```diff
@@ -214,49 +482,120 @@ def __init__(self, app):
         self._timeout = app.options.get('httpTimeout')
 
     def get_android_app_metadata(self, app_id):
-        """Retrieves detailed information about an Android App."""
+        return self._get_app_metadata(
+            platform_resource_name=_ProjectManagementService.ANDROID_APPS_RESOURCE_NAME,
```

Comment:
platform_resource_name is a mouthful, but I don't have a shorter suggestion either :)

Reply:
Acknowledged. Descriptiveness and brevity are always in tension when it comes to readability; here I favour descriptiveness due to the slightly complex list of parameters we need to pass into the helper methods common to both Android and iOS.